src/library/dao/playlistdao.h

```diff
@@ -144,6 +144,7 @@ class PlaylistDAO : public QObject, public virtual DAO {
     QMultiHash<TrackId, int> m_playlistsTrackIsIn;
     AutoDJProcessor* m_pAutoDJProcessor;
     DISALLOW_COPY_AND_ASSIGN(PlaylistDAO);
+    void createTableIDList(QList<int>* pIdList, int playListID);
```

Comment:
QList<int> getPlaylistTrackIds(int playlistId);

"Playlist...", because "TrackId" is a actually different concept. Unfortunately nobody came up with a dedicated name for playlist entries.